huaweicloud/resource_huaweicloud_nat_gateway_v2.go

```diff
@@ -170,6 +194,12 @@ func resourceNatGatewayV2Read(d *schema.ResourceData, meta interface{}) error {
 	d.Set("region", GetRegion(d, config))
 	d.Set("enterprise_project_id", natGateway.EnterpriseProjectID)
 
+	gatewayTags, err := tags.Get(networkClient, "nat_gateways", d.Id()).Extract()
+	if err != nil {
+		return fmtp.Errorf("error getting gateway tags: %s", err)
```

Comment:
We should check if the API is registered in all regions

Reply:
fixed.